en/log/ulog_file_format.md

```diff
@@ -1,15 +1,17 @@
 # ULog File Format
 
 ULog is the file format used for logging system data.
-The format is self-describing, i.e. it contains the format and message types that are logged.
 
-It can be used for logging device inputs (sensors, etc.), internal states (cpu load, attitude, etc.) and printf log messages.
+The format is self-describing, i.e. it contains the format and message types that are logged (note that the [system logger](../log/logging.md) does allow the *default set* of logged topics in `Logger::add_default_topics()` to be replaced from an SD card).
```

Comment:
Since this page is about the file format, I would not point to a specific method implementation here (`Logger::add_default_topics()`).
I'd rather keep a generic note with link to `../log/logging.md` for users who want to use/customize logging.

Reply:
Thanks @bkueng . Fixed up and merged.